src/openms/thirdparty/evergreen/src/Utility/graph_to_dot.hpp

```diff
@@ -2,6 +2,8 @@
 #define _GRAPH_TO_DOT_HPP
 
 #include <fstream>
+#include <map>
```

Comment:
What happened here?

Reply:
What do you mean?

Reply:
I think he means that the thidparty library should not have any changes regarding the map

Reply:
I've changed it back now.